### ISSUE TITLE
`Build` on context menu not working

### ISSUE THREAD

**Issue by armanio123** [microsoft/nodejstools#2212]
##### Expected 
Command on context menu `Build` builds .ts files.

##### Actual Behavior 
- NTVS Version: 1.5.10717.1 Commit Hash:c892fc71536f0cf66cc3c34359ae0b7e86fc0d6d
- Visual Studio Version: 16.3.0 Preview 2.0
- Node.js Version: 10.4.1

##### Steps to Reproduce
1. Create a new folder on the file explorer.
2. Open Visual Studio > File > Open > Folder
3. Choose the newly created folder.
4. Add a `tsconfig.json` file.
5. Right click the tsconfig.json > Build.
6. See error on output Tsc.

```
=== Build Started: C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.6\tsc.exe -p "C:\Users\user\source\repos\OpenFolderError\tsconfig.json" ===
System.ComponentModel.Win32Exception (0x80004005): The system cannot find the file specified
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start()
   at Microsoft.VisualStudioTools.Project.ProcessOutput..ctor(Process process, Redirector redirector)
Error - Failed to start 'C:\Program Files (x86)\Microsoft SDKs\TypeScript\3.6\tsc.exe'
```

![image](https://user-images.githubusercontent.com/13305542/62250226-3877e980-b3a1-11e9-9226-de6d7db3a338.png)

**Comment by dalek09** [comment]
TLDR: can someone tell me, where is the script that is running the build operation for Typescript (as a Folder NOT a .njsproj Project instance) in Visual Studio 2019 Version 16.9.0 (the current latest build)?

I have a very similar issue so I've done some digging and you are correct, tsc.exe no longer exists (Typescriopt V3.2 and later); instead the intention seems to be to run via tsc.cmd which, depending on the folder being launched from (i.e. does it have a node.exe, which it won't in this case), runs node.exe passing in a file to be run of "./node_modules/typesctript/bin/tsc" and the current file to be compiled as a parameter to that. In other words, the tsc compiler is intended to be run as something like "node ./pathTo/tsc.js yourFileToCompile".

Visual studio, however specifically passes C:\Program Files (x86)\Microsoft SDKs\TypeScript\4.2\tsc.exe which, of course, is not there.

I tried copying the C:\Program Files (x86)\Microsoft SDKs\TypeScript\4.2\tsc.cmd as C:\Program Files (x86)\Microsoft SDKs\TypeScript\4.2\tsc.exe.cmd hoping that would work but it doesn't and I don't understand why. The only thing is that the command line exeecuted seems to have no parentheses around the tsc.exe command and so when I try from a command promt it tries to run C:\Program, which of course is no a runnable program.

Can someone please respond as this issue is stil extant after 18 months!
